client-linuxapp/src/serviceinfo.rs

```diff
@@ -606,6 +623,7 @@ async fn process_serviceinfo_in(si_in: &ServiceInfo, si_out: &mut ServiceInfo) -
         if sshkey_user.is_none() || sshkey_key.is_none() {
             bail!("SSHkey module missing username or key");
         }
+        create_user(sshkey_user.as_ref().unwrap()).context("Error creating user")?;
```

Comment:
there are now 2 contexes with the very same text, so either it adds more information or I'd remove the one within `create_user`